src/tests/whitespace_test

```diff
@@ -35,3 +35,14 @@ fi
             exit found
         }
     "
+
+declare found_file=false
+while read file; do
+    [[ $file == "src/config/testconfigs/noparse.api.conf" ]] && continue
+    [[ $file =~ ^src/tests/cmocka/p11_nssdb/.*db ]] && continue
+    test `tail -c 1 $ABS_TOP_SRCDIR/$file` && \
+        echo "Missing new line at the eof: $file" && \
+        found_file=true
+done < <(git ls-files)
+
+exit `! $found_file`
```

Comment:
This won't actually substitute the exit status. The expansion of back-ticks will be empty string.